client/post-editor/editor-notice/index.jsx

```diff
@@ -117,6 +142,10 @@ export class EditorNotice extends Component {
 				} );
 
 			case 'view':
+				if ( 'scheduled' === this.props.message ) {
```

Comment:
These messages are a bit arbitrarily named anyways. Do you think it'd make sense to just create a separate `preview` message label?